### PR TITLE
Present User Selected Time and Now in UTC format

### PR DESCRIPTION
**Pull request by mengyaoyang11**
Problem statement:
Currently clicking the "Now" button sends the local time now to the query, not the UTC time, since there is a time difference between local time(either in the US or in India) and UTC, if the sloop instance is not running during that time period, we will get an empty screen after clicking the 'Now' button. This is expected behavior, customer who keeps running sloop for a long time will not have this issue.

Solution:
Display both customer selected time and now in UTC format

### CODE REVIEW

File: pkg/sloop/webserver/webfiles/sloop_ui.js [hunk 1]
```diff
@@ -532,13 +533,13 @@ $(document).ready(function() {
         && ! Date.parse(sessionStorage.getItem('setSelectedEndTime')) < new Date(userDate.getTime() - 3000)){
         userDate = new Date(sessionStorage.getItem('selectedEndTime'));
     }
-    userDate.setMinutes(now.getMinutes() - now.getTimezoneOffset());
+    userDate.setMinutes(userDate.getMinutes() - userDate.getTimezoneOffset());
```
Comment (duke-harlan) [inline pkg/sloop/webserver/webfiles/sloop_ui.js:536]: does `.getUTCMinutes()` (as seen line 542) work here?

**Review by duke-harlan — APPROVED** [review]
it looks like we're using 2 techniques to accomplish the same change - if they produce the same behavior, lets stick with just one

File: pkg/sloop/webserver/webfiles/sloop_ui.js (hunk 1 above)
Comment (mengyaoyang11) [reply pkg/sloop/webserver/webfiles/sloop_ui.js:536]: Tried `.getUTCMinutes()` here as well but not producing the same behavior. Will talk to Sana to see if the change is needed or not

Comment (mengyaoyang11) [reply pkg/sloop/webserver/webfiles/sloop_ui.js:536]: @duke-harlan Updated the pr to address the comment: use one technique to present both user selected time and now to local time. Please review again

**Review by duke-harlan — DISMISSED** [review]

**Review by sana-jawad — CHANGES_REQUESTED** [review]
Please make sure the same time zone is used across.

**Review by sana-jawad — APPROVED** [review]

**Pull request merged**